src/components/Player/Pages/Matches/Matches.css

```diff
@@ -0,0 +1,16 @@
+@media only screen and (max-width: 880px) {
+  th:nth-child(4), td:nth-child(4),
+  th:nth-child(5), td:nth-child(5),
+  th:nth-child(8), td:nth-child(8),
+  th:nth-child(9), td:nth-child(9),
+  th:nth-child(10), td:nth-child(10) {
+    display: none;
```

Comment:
We could select a range of elements using something like [css-to-select-range-of-children](https://gist.github.com/rondevera/167627). Here's a fiddle http://jsfiddle.net/46XRe/